token-metadata/program/Cargo.toml

```diff
@@ -8,6 +8,8 @@ license = "AGPL-3.0"
 edition = "2021"
 readme = "README.md"
 
+[profile.release]
+overflow-checks = true
```

Comment:
Does this have an effect when running in `ebpf` VM?
I thought I read something somewhere that it doesn't .. not 100% sure tough

Reply:
That is a good question. I'll look into it.

Reply:
Obviously now I cannot find anything about this .. so maybe it's not an issue at all :)

Reply:
Seems to be used by other Solana devs: https://github.com/coral-xyz/anchor/pull/1758